pom.xml

```diff
@@ -522,6 +522,9 @@
                 change to fix a integer overflow issue.
                 TODO: remove this after Parquet 1.13 release -->
               <exclude>org.apache.parquet.column.values.dictionary.DictionaryValuesWriter#dictionaryByteSize</exclude>
+              <exclude>org.apache.parquet.hadoop.util.ColumnMasker.MaskMode</exclude>
+              <exclude>org.apache.parquet.hadoop.util.CompressionConverter</exclude>
+              <exclude>org.apache.parquet.hadoop.util.ColumnMasker</exclude>
```

Comment:
You've accepted my previous comment but nothing changed.

> (...) if the `util` package is not for public use than we shall add `@Private` to all of its classes and maybe also add a `package-info.java` stating that this is not for public use and add similar comments here as well. Otherwise, we should maintain backward compatibility for these classes as well.

Reply:
@gszadovszky, Sorry my bad. Making it private won't work because it is still used outside util package. So I just revert the refactoring code to avoid the exclusion here. We can do it later.